scripts/ingests/Y_dwarf_source_ingest.py

```diff
@@ -0,0 +1,42 @@
+# Script to ingest Y dwarfs from Kirkpartick+2019
+
+from astrodbkit2.astrodb import create_database
+from astrodbkit2.astrodb import Database
+#from simple.schema import *
+from astropy.table import Table
+
+connection_string = 'sqlite:///../../SIMPLE.db'  # SQLite
+create_database(connection_string)
+db = Database(connection_string)
+db.load_database('../../data')
+
+
+# load table of sources to ingest
+Ydwarfs = Table.read('Y-dwarf_table.csv',data_start=2)
+
+# find sources already in database
+existing_sources = []
+missing_sources = []
+for i,name in enumerate(Ydwarfs['source']):
+	if len(db.search_object(name,resolve_simbad=True)) != 0:
+		existing_sources.append(i)
+	else:
+		missing_sources.append(i)
+
+# add missing references
+temp = db.query(db.Publications.c.name).all()
```

Comment:
As mentioned during the meeting, here is a more compact way of doing most of lines 27-33:
```
ref_list = Ydwarfs['reference'].tolist()
included_ref = db.query(db.Publications.c.name).filter(db.Publications.c.name.in_(ref_list)).all()
included_ref = [s[0] for s in included_ref]
new_ref = list(set(ref_list)-set(included_ref))
```
The first line gets a list of the sources from your csv file.
The second line gets all database entries that match any in your list
The third line is just some quick formatting of the result list
The fourth line takes the difference: it outputs all references from ref_list that are not among those returned by the database query.

Reply:
Actually, I guess an extra step would be needed to convert it to a dictionary:
`new_ref = [{'name': s} for s in new_ref]`